packages/appium/docs/en/guides/migrating-1-to-2.md

```diff
@@ -60,6 +60,16 @@ With Appium 1.x, command-line options specific to particular drivers were all ho
 
 With Appium 2.x, all driver- and platform-specific CLI params have been moved to the drivers themselves. To access them, you'll now need to prepend the argument with the extension type (either `driver` or `plugin`) and the name of the extension. For example, `--chromedriver-executable` becomes `--driver-uiautomator2-chromedriver-executable`.
 
+### :bangbang: Driver-specific automation commands
+
+The definition of certain commands that pertain only to specific drivers has been moved to those
```

Comment:
this is hard-wrapped and other stuff ain't

Reply:
yeah i'm kinda hard wrapping as i go. didn't want to create a diff for the whole file in this PR